src/components/Sections/hooks/useKonnectorsByCat.ts

```diff
@@ -104,33 +76,41 @@ export const useKonnectorsByCat = (): Section[] => {
       : []
   }, [suggestedKonnectorsQuery.data, candidatesSlugBlacklist])
 
+  const [groupedData, setGroupedData] =
+    useState<{ [key: string]: IOCozyKonnector[] }>()
+
   useEffect(() => {
     const fetchData = async (): Promise<void> => {
       if (!client) return
       const grouped = await fetchAllKonnectors(client)
       setGroupedData(grouped)
     }
-
     void fetchData()
   }, [client])
 
-  const sortedData = useMemo(
-    () =>
+  const sortedData = useCallback(
```

Comment:
I'm curious why do you changed the useMemo to a useCallback ?

Reply:
I'm unsure myself how to negate any performance issue in this cumbersome formatting flow. I was testing alternatives. In a general way I see useMemo as a last resort tool

Reply:
For me, making a useCallback + call is the same as making a useMemo. I find the useMemo more readable but I haven't tested the difference in performance between the two.